src/Data/Unit/Polymorphic.agda

```diff
@@ -0,0 +1,38 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- The universe polymorphic unit type and the total relation on unit
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Unit.Polymorphic where
+
+open import Level
+open import Relation.Nullary
+open import Relation.Binary
+open import Relation.Binary.PropositionalEquality
+
+------------------------------------------------------------------------
+-- A unit type defined as a record type
+
+record ⊤ {ℓ : Level} : Set ℓ where
+  constructor tt
+
+------------------------------------------------------------------------
+-- An ordering relation over the unit type
+
+record _≤_ {ℓ ℓ′ e : Level} (x : ⊤ {ℓ}) (y : ⊤ {ℓ′}) : Set e where
```

Comment:
Interesting that you need the relation to live at its own level. I guess that makes sense as it can live below the levels of its arguments...

Reply:
Yep.  Inspired by the same possibility in categories.